message-index/messages/GHC-64725/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: User-defined type error
+summary: A constraint or type family reduced to a custom type error.
+introduced: 8.0.1
```

Comment:
[`GHC.TypeLits.TypeError`](https://hackage.haskell.org/package/base-4.16.1.0/docs/GHC-TypeLits.html#t:TypeError) was introduced in `base-4.9.0.0`, which came out with [GHC `8.0.1`](https://downloads.haskell.org/~ghc/8.0.2/docs/html/users_guide/8.0.1-notes.html#base)

Reply:
ah yes, I thought I'd save some work for @david-christiansen since I knew it was from 8.0.1

Reply:
That's right - the field is for the code, and I should say something about that on the text of the homepage (#172)

Reply:
```suggestion
introduced: 9.6.1
```

Reply:
Ah, I understand now, thanks!